main.py

```diff
@@ -72,9 +73,14 @@ def main():
         barcodes_str = "','".join(
             pc_reserve_df['barcode'].to_string(index=False).split())
         barcodes_str = "'" + barcodes_str + "'"
+        sierra_query = build_sierra_query(barcodes_str)
         sierra_client.connect()
-        sierra_raw_data = sierra_client.execute_query(
-            build_sierra_query(barcodes_str))
+        try:
+            sierra_raw_data = sierra_client.execute_query(sierra_query)
+        except PostgreSQLClientError:
+            logger.info('Attempting Sierra query again')
+            sierra_raw_data = sierra_client.execute_query(sierra_query)
```

Comment:
should there be another try/except here, or a (bounded) loop?

Reply:
I only want it to retry once -- if it fails the second time that likely indicates a real error and I don't want it to consume too many Sierra resources.